scripts/campaigns.json

```diff
@@ -40,7 +40,7 @@
     {
         "name": "Mr Blinky Bling DEFCON 26 Indie Badge",
         "description": "We're back and this time we're making a social game using blockchain (yes, really!) and badges! Shall we play a game? So we've been trolling you all on Twitter about this whole Blockchain badge concept and the general reaction has been: They must be trolling!",
-        "logoLink": "http://mrblinkybling.com/img/profile.png",
+        "logoLink": "https://www.google.com/images/errors/robot.png",
         "websiteLink": "http://mrblinkybling.com/",
```

Comment:
Also need to change the website link.
Is the website link being validated?

Reply:
We are validating if the website link is in correct format or not

Reply:
http://mrblinkybling.com/ <- this is not a secure website, need to be httpS instead of http.